### ISSUE TITLE
Option 'user' is not known for Git fetcher

### ISSUE THREAD

**Issue by skycaptain** [priv-kweihmann/oelint-adv#219]
In one of my recipes using

```bitbake
SRC_URI = "git://spamhameggs.git;protocol=ssh;user=git;branch=${SRCBRANCH}"
```

I'm getting this warning with latest oelint-adv:

```
[...]:warning:oelint.vars.srcurioptions:Option 'user' is not known with this fetcher type
```

While `user` is [not officially documented](https://www.yoctoproject.org/docs/latest/bitbake-user-manual/bitbake-user-manual.html#git-fetcher), it is allowed to be used: https://github.com/openembedded/bitbake/blob/74042b5b89d5a170013fc1a327ce3a6530fbf7d5/lib/bb/fetch2/git.py#L650. One way to circumvent this is using `git@` in the URL, however I'm leaving this here and report to bitbake mailinglist.

**Comment by priv-kweihmann** [comment]
Yes please - I personally would keep it like it is, unless it becomes "officially" documented
